shopfloor_mobile/static/wms/src/scenario/checkout.js

```diff
@@ -283,15 +282,6 @@ const Checkout = {
                 },
             };
         },
-        select_delivery_packaging_manual_select_options: function () {
-            return {
-                showActions: false,
-                list_item_options: {
-                    loud_title: true,
-                    fields: [{path: "packaging_type"}],
```

Comment:
Can't you simply remove the fields from here instead of dropping the method?

Reply:
I've now removed the list_item_options entirely, as loud_title gave it an ugly look.